### ISSUE TITLE
prefix level never got the AMENT_PYTHON_EXECUTABLE fixes

### ISSUE THREAD

**Issue by BrannonKing** [ament/ament_package#62]
A year ago some fixes were made to allow AMENT_PYTHON_EXECUTABLE to work. However, this never made it into the prefix_level template (it went only into isolated_prefix_level). Even in reviewing the old fix, I'm not sure it should work since it doesn't use the `!...!` operator when it reads the local variable. Essentially, isolated_prefix_level/local_setup.bat.in should be modified such that `%_ament_python_executable%` becomes `!_ament_python_executable!`. The file prefix_level/local_setup.bat.in needs that change in addition to the leading underscore as the variable names are not case sensitive.

Also, I don't think that all the python projects are getting installed with the right python path on Windows regardless of these fixes.

**Comment by BrannonKing** [comment]
PS The current chocolatey python installations go to a different folder that 3.5 did a few months ago.

**Comment by mikaelarguedas** [comment]
> PS The current chocolatey python installations go to a different folder that 3.5 did a few months ago.

Thanks for pointing it out, is the new location matching the one currently documented on the [installation instructions](https://github.com/ros2/ros2/wiki/Windows-Development-Setup#extra-stuff-for-debug-mode) ? `C:\Python36`

**Comment by BrannonKing** [comment]
Yes, Python36 is correct; it appears that my build servers haven't been updated. I just wish that they didn't have to match my local machine and vice-versa.

**Comment by dirk-thomas** [comment]
> A year ago some fixes were made to allow AMENT_PYTHON_EXECUTABLE to work

Can you please reference them to make clear which ones you are referring to.

> Essentially, isolated_prefix_level/local_setup.bat.in should be modified such that `%_ament_python_executable%` becomes `!_ament_python_executable!`.

Can you make a PR with the proposed change.

Can you please describe a set of steps to reproduce the case you are describing.
